xarray/core/dataarray.py

```diff
@@ -1431,6 +1431,12 @@ def isel(
         Dataset.isel
         DataArray.sel
 
+        :doc:`xarray-tutorial:intermediate/indexing/indexing`
```

Comment:
:( This seems to have messed up some formatting though:
<img width="501" alt="image" src="https://github.com/pydata/xarray/assets/2448579/8f274efb-46c7-489e-8676-ceefb9c851d1">

while on `main`
<img width="476" alt="image" src="https://github.com/pydata/xarray/assets/2448579/05a48127-663f-42e9-b580-0f84f9cfb6e2">

Reply:
this should be valid, though. So we might need to fix `napoleon` to allow this.

Reply:
it appears we had a similar issue more than two years ago when we first introduced `:doc:` links to `See Also` (see #5176), where we concluded that this is a bug in `napoleon` (nobody got around to reporting / fixing that since then, though).

Reply:
OK well this is an improvement in any case. Merging.